QuantConnect.TDAmeritrade/TDAmeritradeBrokerage.WebSocket.cs

```diff
@@ -0,0 +1,584 @@
+
+        private void ParseAccountActivity(JToken content)
+        {
+            var accountActivityData = content.ToObject<List<AccountActivityResponseModel>>()?.First();
+
+            if (!accountActivityData.HasValue)
+                return;
+
+            switch (accountActivityData.Value.MessageType)
+            {
+                case "SUBSCRIBED":
+                    Log.Trace($"TDAmeritradeBrokerage:DataQueueHandler:OnMessage:AccountAcctivity: subscribed successfully, Description: {accountActivityData.Value.MessageData}");
+                    break;
+                case "ERROR":
+                    Log.Error($"TDAmeritradeBrokerage:DataQueueHandler:OnMessage:AccountAcctivity: not subscribed, Description: {accountActivityData.Value.MessageData}");
+                    break;
+                case "BrokenTrade": // After an order was filled, the trade is reversed or "Broken" and the order is changed to Canceled.
+                    break;
+                case "ManualExecution": // The order is manually entered (and filled) by the broker.  Usually due to some system issue.
+                    break;
+                case "OrderActivation": // A Stop order has been Activated
+                    break;
+                case "OrderCancelReplaceRequest": // A request to modify an order (Cancel/Replace) has been received (You will also get a UROUT for the original order)
+                    break;
+                case "OrderCancelRequest": // A request to cancel an order has been received
+                    var candelOrder = DeserializeXMLExecutionResponse<OrderCancelRequestMessage>(accountActivityData.Value.MessageData);
+                    HandleOrderCancelRequest(candelOrder);
+                    break;
+                case "OrderEntryRequest": // A new order has been submitted
+                    var newOrder = DeserializeXMLExecutionResponse<OrderEntryRequestMessage>(accountActivityData.Value.MessageData);
+                    HandleSubmitNewOrder(newOrder);
+                    break;
+                case "OrderFill": // An order has been completely filled
+                    var orderFill = DeserializeXMLExecutionResponse<OrderFillMessage>(accountActivityData.Value.MessageData);
+                    HandleOrderFill(orderFill);
+                    break;
+                case "OrderPartialFill": // An order has been partial filled
+                    break;
+                case "OrderRejection": // An order was rejected
+                    break;
+                case "TooLateToCancel": // A request to cancel an order has been received but the order cannot be canceled either because it was already canceled, filled, or for some other reason
+                    break;
+                case "UROUT": // Indicates "You Are Out" - that the order has been canceled
+                    DeserializeXMLExecutionResponse<UROUTMessage>(accountActivityData.Value.MessageData);
+                    break;
+            }
```

Comment:
I removed extra `case`, because I don't catch one of them except ones you can see.
For example: `UROUT` and `OrderFill` - return the same infromation about order